selfdrive/car/hyundai/carstate.py

```diff
@@ -154,17 +155,22 @@ def update(self, cp, cp_cam):
   def update_canfd(self, cp, cp_cam):
     ret = car.CarState.new_message()
 
-    if self.CP.carFingerprint in EV_CAR:
-      ret.gas = cp.vl["ACCELERATOR"]["ACCELERATOR_PEDAL"] / 255.
-    elif self.CP.carFingerprint in HYBRID_CAR:
-      ret.gas = cp.vl["ACCELERATOR_ALT"]["ACCELERATOR_PEDAL"] / 1023.
-    ret.gasPressed = ret.gas > 1e-5
-    ret.brakePressed = cp.vl["BRAKE"]["BRAKE_PRESSED"] == 1
+    if self.CP.carFingerprint in (EV_CAR | HYBRID_CAR):
+      if self.CP.carFingerprint in EV_CAR:
+        ret.gas = cp.vl["ACCELERATOR"]["ACCELERATOR_PEDAL"] / 255.
+      else:
+        ret.gas = cp.vl["ACCELERATOR_ALT"]["ACCELERATOR_PEDAL"] / 1023.
+      ret.gasPressed = ret.gas > 1e-5
+    else:
+      ret.gasPressed = bool(cp.vl["ACCELERATOR_BRAKE_ALT"]["ACCELERATOR_PEDAL_PRESSED"])
+
+    brake_msg = "ACCELERATOR_BRAKE_ALT" if self.CP.carFingerprint not in (EV_CAR | HYBRID_CAR) else "BRAKE"
+    ret.brakePressed = cp.vl[brake_msg]["BRAKE_PRESSED"] == 1
```

Comment:
Can you use the new common brake signal we merged in https://github.com/commaai/openpilot/pull/26170?

Reply:
@sunnyhaibin i'll take care of the rest of this PR. i want to do some panda cleanup first. the addr check stuff is getting really messy now

Reply:
@adeebshihadeh Thanks! Let me know if you need anything.